app/pages/admin/workflow-default-dialog.spec.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+import WorkflowDefaultDialog from './workflow-default-dialog';
+
+describe('WorkflowDefaultDialog', () => {
+  let wrapper;
+  let onSuccessSpy;
+  let onCancelSpy;
+
+  before(() => {
+    onSuccessSpy = sinon.spy();
+    onCancelSpy = sinon.spy();
+    wrapper = shallow(<WorkflowDefaultDialog onSuccess={onSuccessSpy} onCancel={onCancelSpy}/>);
+  });
+
+  it('renders without crashing', () => {
+    const WorkflowDefaultDialogContainer = wrapper.find('div').first();
+    assert.equal(WorkflowDefaultDialogContainer.length, 1);
+  });
+
+  it('calls the onSuccess handler', () => {
+    wrapper.find('button#workflowDefaultDialogSuccess').simulate('click');
```

Comment:
I'd prefer not adding CSS ids or classes to elements just for the purposes of testing. Since there are only two buttons using enzyme's `first()` or `last()` is fine or [`at()`](http://airbnb.io/enzyme/docs/api/ShallowWrapper/at.html)